spec/unit/functions/lookup_spec.rb

```diff
@@ -1122,6 +1122,61 @@ def explain(key, options = {})
       end
     end
 
+    context 'and an environment Hiera v5 configuration using uris' do
+      let(:env_hiera_yaml) do
+        <<-YAML.unindent
+        ---
+        version: 5
+        hierarchy:
+          - name: Globs
+            uris:
+              - "http://test.example.com"
+              - "/some/arbitrary/path"
+              - "urn:with:opaque:path"
```

Comment:
one example in the ticket was something with a space and a flag like `"dothis -f bar"` - think we should include a whitespace and flag example

Reply:
In that case it must be listed as another example of a bad URI. There's no way to create an URI from that string, partial or otherwise.